src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_actions.py

```diff
@@ -97,6 +97,8 @@ def _handle_vm_nics(namespace):
     namespace.network_interface_ids = nics
     namespace.network_interface_type = 'existing'
 
+    namespace.public_ip_address_type = 'none'
```

Comment:
yes, this is the established pattern for all folded args

---
In reply to: [73382149](https://github.com/Azure/azure-cli/pull/599#discussion_r73382149) [](ancestors = 73382149)